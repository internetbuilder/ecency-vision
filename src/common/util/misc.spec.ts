import {parseUrl} from "./misc";

it('1 - Invalid', () => {
    expect(parseUrl("foo")).toMatchSnapshot();
    expect(parseUrl("")).toMatchSnapshot();
    expect(parseUrl(" foo  https://ecency.com bar ")).toMatchSnapshot();
});


it('2 - Valid', () => {
    expect(parseUrl("https://ecency.com")).toMatchSnapshot();
    expect(parseUrl("  https://ecency.com  ")).toMatchSnapshot();
    expect(parseUrl("https://weebecash.ngrok.io/hive-125125/@ecency/onboarding-more-users-join-us")).toMatchSnapshot();
});
