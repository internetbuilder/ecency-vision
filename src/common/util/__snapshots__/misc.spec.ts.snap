// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1 - Invalid 1`] = `null`;

exports[`1 - Invalid 2`] = `null`;

exports[`1 - Invalid 3`] = `null`;

exports[`2 - Valid 1`] = `"https://weebecash.ngrok.io/"`;

exports[`2 - Valid 2`] = `"https://weebecash.ngrok.io/"`;

exports[`2 - Valid 3`] = `"https://weebecash.ngrok.io/hive-125125/@ecency/onboarding-more-users-join-us"`;
